unfoldingWord/translationCore
# Project import QA issues

#### This pull request addresses:

Address multiple QA issues with online project import.

#### How to test this pull request:
1. Previous search results are displayed if the current user does not have any projects on Door43.
To replicate:
 - Log in with a username that has no projects and go to import online projects. A message is displayed correctly indicating that no projects were found.
 - Enter a username who has some projects. Import one of the projects or just cancel.
 - Go back to import online projects. You'll see that the username is now correctly switched to the logged in user, but the previous result set it displayed.
 - The same thing happens if you have a result set and then change the username in the search to one who has no projects.

2. If the using the Internet warning is displayed after clicking the Import button, the "Importing..." message is not displayed after dismissing the warning.

3. If the import fails because the project already exists when I go back to import online projects the previously selected project is still selected.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2414)
